packages/core/src/destination-kit/index.ts

```diff
@@ -94,12 +94,15 @@ export type AudienceResult = {
 }
 
 export type AudienceMode = { type: 'realtime' } | { type: 'synced'; full_audience_sync: boolean }
+export type Personas = { computation_id: 'string'; computation_key: 'string'; [key: string]: any }
```

Comment:
nit: Should you use unknown here? You'll probably see a warning in your editor as well.
```suggestion
export type Personas = { computation_id: 'string'; computation_key: 'string'; [key: string]: unknown }
```